llvm/tools/llvm-mca/llvm-mca.cpp

```diff
@@ -364,6 +405,8 @@ int main(int argc, char **argv) {
 
   cl::HideUnrelatedOptions({&ToolOptions, &ViewOptions});
 
+  InstructionTablesOption.setValueExpectedFlag(cl::ValueOptional);
```

Comment:
please just add `cl::ValueOptional` in the declaration of `InstructionTablesOption`